hack/patch_node.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash -e
+
+PROXY_PORT=9999
+
+function finish {
+  rc=$?
+  if jobs 1 | grep -q Running; then
+    echo
+    echo "Killing proxy (pid=$proxy_pid)..."
+    kill "$proxy_pid"
+  fi
+  exit "$rc"
+}
+
+# creates/removes prow/sriov resources for the given $node
```

Comment:
Maybe it worth mentioning in the comment that this resource will actually effect the amount of PFs delegated to the cluster.

Reply:
It doesn't affect the number of PFs delegated, it just affect how many jobs are running in the same time.
and this is determined becaused the request/limit is added to the job as explained in the README.
k8s will allocate the requested amount of prow/sriov, if it has enough free ones, else the job would be pending.

The CNI is responsible to allocate 1 PF, calling the CNI twice will allocate 2 PFs (explained in the README).

Here its only create the k8s resources.

Reply:
ohhh. OK got it.
Thanks.